hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/PrisonerSynchroniserService.kt

```diff
@@ -34,9 +36,8 @@ class PrisonerSynchroniserService(
       restrictedPatientData = restrictedPatient,
     )
     // only save to open search if we encounter any differences
-    if (prisonerDifferenceService.prisonerHasChanged(existingPrisoner, prisoner)) {
+    if (prisonerDifferenceService.hasChanged(existingPrisoner, prisoner)) {
```

Comment:
I renamed prisonerHasChanged as I expected to use it for objects other than prisoner like CurrentIncentive